src/tests/ftest/util/osa_utils.py

```diff
@@ -290,6 +291,22 @@ def set_cont_class_properties(self, oclass="S1"):
             self.ior_cmd.dfs_dir_oclass.update(None, "ior.dfs_dir_oclass")
             self.container.oclass.update(None)
 
+    def get_random_test_ranks(self, total_ranks=2, stop_individually=True):
```

Comment:
In my opinion, `stop_individually` isn't a good variable name because it's used as a flag and we're not stopping anything in this method. It's better to make methods de-coupled from other code. What if you want to use this method for `dmg system start` in the future? Maybe use `comma_separated`?

Reply:
Change the parameter name to stop_individual_ranks.